tests/foreman/api/test_capsule.py

```diff
@@ -0,0 +1,227 @@
+"""Tests for the ``smart_proxies`` paths.
+
+:Requirement: Smartproxy
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Capsule
+
+:Team: Endeavour
+
+:TestType: Functional
+
+:CaseImportance: Critical
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+from fauxfactory import gen_url
+from requests import HTTPError
+
+from robottelo.config import user_nailgun_config
+
+
+pytestmark = [pytest.mark.run_in_one_thread]
+
+
+@pytest.mark.e2e
+@pytest.mark.upgrade
```

Comment:
I think this tests is not a very good candidate fr e2e or upgrade as we are not checking any e2e functionality.


E2e cases for capsule should have everything like capsule sync, host reg etc.

Reply:
noted, but it is the best we have right now, until we unstub the one below